docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
 -r ../dev-requirements.txt
 sphinx!=3.0.0
 requests>=2,<2.16
-furo
+furo==2020.9.8b4
```

Comment:
If you remove the pin, I swear I'll be careful about my version numbers. Plus, there's automation now to prevent me from derping like this again. :3

Reply:
Was letting an alpha project be wild and free ;) I can remove the pin.